modules/cij/__init__.py

```diff
@@ -81,6 +81,7 @@ def err(txt, file=sys.stdout):
     print("%s# %s%s%s" % (PR_ERR_CC, get_time_stamp(), txt, PR_NC), file=file)
     _flush(file)
 
+
```

Comment:
This is probably nit-picking, but these should go into the first commit as that is where the spaces where removed in the first-place :)